app/operations/rating_books/create.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module RatingBooks
+  class Create < ::Callable
+    def initialize(book:, user:, rating:)
+      @book = book
+      @user = user
+      @rating = rating
+    end
+
+    def call
+      user.rating_books.find_by(book_id: book.id).destroy if user.already_rated?(book)
```

Comment:
1. write tests for this
2. use https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert